src/components/Api.js

```diff
@@ -1,19 +1,20 @@
 class Api {
   constructor(options) {
-    (this.baseUrl = options.baseUrl), (this.headers = options.headers);
+    this._baseUrl = options.baseUrl;
+    this._headers = options.headers;
   }
 
-  postPledge({ Name, Surname, Email, Pledge }) {
-    return fetch(`${this.baseUrl}/pledges`, {
+  postPledge({ name, surname, email, pledge }) {
```

Comment:
```suggestion
     postPledge(data) {
    fetch(`${this._baseUrl}/pledges`, {
      method: 'POST',
      headers: this._headers,
      body: JSON.stringify(data)
    })
```
I think this syntax might be the better because we already created data const in the index.js so I don't think there is a point in writing this object all over again.

Reply:
We need to have the body keys capitalized.